snippets/rst.json

```diff
@@ -0,0 +1,115 @@
+{
+    "main title": {
+        "prefix": ["main title", "="],
+        "body": ["============================\n${0}\n============================\n\n"],
+        "description": "Add main title"
+    },
+    "title": {
+        "prefix": ["title", "="],
+        "body": ["${0}\n============================\n\n"],
+        "description": "Add title"
+    },
+    "section": {
+        "prefix": ["section", "-"],
+        "body": ["${0}\n------------\n\n"],
+        "description": "Add section"
+    },
+    "Links": {
+        "prefix": ["l", "link", "hyperlink"],
+        "body": ["${1:text}_\n.. _${1:text} ${2:url}${0}"],
```

Comment:
I am unsure if the snippet jumps to the second placeholder. Would you be able to verify this?